context/app/static/js/components/Detail/Dataset/Dataset.spec.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable camelcase */
+import React from 'react';
+import 'whatwg-fetch';
```

Comment:
Add a comment about why this is needed?

Reply:
Good idea. I think I'm also going to move it into `setupTests.js`.